src/lib/containers/table/SynapseTable.tsx

```diff
@@ -751,9 +791,13 @@ export default class SynapseTable extends React.Component<
       if (showAccessColumn && token) {
         const rowSynapseId = `syn${row.rowId}`
         rowContent.push(
-          <td key={`(${rowIndex},accessColumn)`} style={{ textAlign: 'center' }} className="SRC_noBorderTop">
+          <td
+            key={`(${rowIndex},accessColumn)`}
```

Comment:
I don't believe theres any explicit convention being followed here, other than making the key tuple-like, but this can change, do you think _ is more readable/standard?

Reply:
I don't think this matters one way of the other. It's mostly for React internal rendering/update anyway. I was just curious if there was a convention.  I'm totally fine with leaving it as is